ci-operator/jobs/infra-periodics.yaml

```diff
@@ -144,6 +144,49 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: api.ci
+  cron: 0 1 * * *
+  decorate: true
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-daily-bugzilla-refresh
+  spec:
+    containers:
+    - args:
+      - |-
+        --query=is:pr
+        state:open
+        label:bugzilla/invalid-bug
```

Comment:
Adding label:lgtm might reduce the number even further. I'm not sure all repos used approved, so maybe not that....